auto_gptq/modeling/_base.py

```diff
@@ -187,22 +193,17 @@ def quantize(
         if self.quantize_config.quant_method in QUANTIZE_BLACK_LIST:
             raise ValueError(f"Unsupported quantization operation for quant method: {self.quantize_config.quant_method}")
 
+        # TODO: need to isolate pack code that is causing this regression
         # alert users to limit threads so packing performance does not regress by up to ~100x
-        thread_warning = """If you have not already done so, please inject the following code at the very top of your 
-quantization script so the packing stage is optimized for speed. Using too many cores may reduce packing performance.
-----
-import os
-import math
-max_threads = str(min(8, os.cpu_count()))
-os.environ['OMP_NUM_THREADS'] = max_threads
-os.environ['OPENBLAS_NUM_THREADS'] = max_threads
-os.environ['MKL_NUM_THREADS'] = max_threads
-os.environ['VECLIB_MAXIMUM_THREADS'] = max_threads
-os.environ['NUMEXPR_NUM_THREADS'] = max_threads
-os.environ['NUMEXPR_MAX_THREADS'] = max_threads
-----
-"""
-        logger.warning(thread_warning)
+        if "OPENBLAS_NUM_THREADS" not in os.environ or int(os.environ["OPENBLAS_NUM_THREADS"]) > 8:
+            thread_warning = """If you have not already done so, please inject the following code at the very top of your
+    quantization script so the packing stage is optimized for speed. Using too many cores may reduce packing performance.
+    ----
+    import os
+    os.environ['OPENBLAS_NUM_THREADS'] = str(1)
+    ----
+    """
+            logger.warning(thread_warning)
```

Comment:
@fxmarty  We found issues with this change. Testing right now and should have more correct fix in coming hours.

Reply:
should be fixed with https://github.com/AutoGPTQ/AutoGPTQ/pull/640/commits/a3863a44e246340496d0313b8c57324338e8993d  but need further testing.